packages/ses/README.md

```diff
@@ -137,6 +146,61 @@ c1.globalThis === c2.globalThis; // false
 c1.globalThis.JSON === c2.globalThis.JSON; // true
 ```
 
+The global scope of every compartment includes a shallow, specialized copy of
+the JavaScript intrinsics, omitting `Date.now` and `Math.random` since these
+can be used as covert communication channels between programs that have not
+been expressly introduced.
+However, a compartment may be expressly given access to these objects
+through the first argument to the compartment constructor or by assigning
+them to the compartment's `globalThis` after construction.
+
+```js
+const powerfulCompartment = new Compartment({ Math });
+powerfulCompartment.globalThis.Date = Date;
+```
+
+### Compartment + Lockdown
+
+Compartment and Lockdown combine to isolate client code in an environment with
+limited powers and communication channels.
+A compartment receives only the capabilities it is expressly given and cannot
+modify any of the shared intrinsics.
+To do this, every compartment gets its own globals, including such
```

Comment:
What is "this"? I think it's "isolate code", but you dropped in a sentence between saying that and this. If correct, I'd suggest changing "do this" to "isolate code".